.github/workflows/semantic-release.yml

```
@@ -0,0 +1,47 @@
+          make lint-check
+          make coverage
+
+      - name: Python Semantic Release
+        id: release
+        uses: python-semantic-release/python-semantic-release@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publish package to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        if: steps.release.outputs.released == 'true'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)